server/directory/serializers.py

```diff
@@ -30,9 +29,12 @@ def to_representation(self, instance):
         representation = super().to_representation(instance)
 
         # remove empty fields
-        for field in instance._meta.get_fields():
-            field = field.name
-            if representation[field] is None or representation[field] == {} or representation[field] == [] or representation[field] == '':
-                representation.pop(field)
+        fields_to_remove = []
```

Comment:
could we do something like this? or is this bc we need to return original representation obj?

```py
bad = None, {}, [], ''

return {
  field: value for field, value in representation.items() if value not in bad
}
```